client/dashboard/data/index.ts

```diff
@@ -295,6 +296,13 @@ export const fetchTwoStep = async (): Promise< TwoStep > => {
 	return wpcom.req.get( '/me/two-step' );
 };
 
+export const fetchSiteFeatures = async ( siteIdOrSlug: string ): Promise< SiteFeatures > => {
```

Comment:
Is this really needed. I recall that with @jsnajdr we made an update to the regular site endpoint to include the features there under `plan`

Reply:
You're right, the features are available there! Thanks, updated: https://github.com/Automattic/wp-calypso/pull/103227/commits/f5d8429441e03201d61d044cb254c916ac49b19e